### ISSUE TITLE
Do users notice the "other search tools" option in site-wide results display?

### ISSUE THREAD

**Issue by nickykrause** [fecgov/fec-testing#59]
We attempted to address this question recently in [this script](https://github.com/18F/fec-testing/blob/master/Testing-Scripts/Test-Script_2017-05-12.md), but only 2 users showed up for testing. 

The `other search tools` panel appears to the left of the search results, like so:

![image](https://cloud.githubusercontent.com/assets/1450173/26112493/abf2f4d6-3a1d-11e7-8e77-f932909d19ce.png)

The two users who tested this were working on a task to find individual contributions. Specifically, they were asked: "If you were wanted to know who one of your neighbors or family members (or any specific person) donated money to in the last presidential election, what would you do to find out?" 

If users did not naturally use the site-wide search, they were eventually prompted to do so. In this case, the first user naturally used site-wide search, and he did _not_ notice the `other search tools`. He failed the task and had to have the `individual contributions` link pointed out to him.

The second user said her natural inclination would be to go to the Campaign finance data menu option to complete this task, but she was prompted to try the search bar. She _did_ notice the `individual contributions` option in the `other search tools` menu.

**In summary, with this sample of users, 1/2 of them noticed the `other tools` menu alongside the search results. We need more data on this.**
